app/main.ipynb

```diff
@@ -126,17 +158,26 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
+   "execution_count": 16,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "27.0"
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
     "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "array[5:7].sum() - 0.1"
```

Comment:
You should use multi-indexing you should summarize a subsequence of an array and should not use mathematical operations